packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -383,4 +429,38 @@ export class SingleAddressWallet implements ObservableWallet {
       )
     );
   }
+
+  // TODO - maybe we can move this to keyAgent? + cleanup
+  async #constructVotingAuxData(props: VotingAuxDataProps): Promise<Cardano.AuxiliaryData> {
```

Comment:
How about moving this and voting constants under `wallet/Transaction`? This function can take in a KeyAgent as a dependency argument.